packages/react/src/field/Field.tsx

```diff
@@ -15,7 +15,7 @@ export type FieldProps = {
 	invalid?: boolean;
 	/** Describes the purpose of the field. */
 	label: string;
-	/** Override the default secondary label. */
+	/** Text to prepend to the default secondary label. */
 	secondaryLabel?: string;
```

Comment:
This small change keeps the logic for secondary labels in the same location.

Rather than just the text overriding the default secondary label ('optional'), we now append any text to the default secondary label. 

Note: Date picker is the only component using this feature of the `Field` component.